packages/fx-core/src/plugins/solution/fx-solution/commonQuestions.ts

```diff
@@ -314,6 +351,10 @@ async function askCommonQuestions(
     );
   }
 
+  const m365TenantResult = await checkM365Tenant(ctx.envInfo, appstudioTokenJson);
```

Comment:
Should we use `ctx.solutionContext.envInfo` rather than `ctx.envInfo` currently? Because ctx.envInfo is for v2, and v2 is not ready yet.

Reply:
OK, I was wrong.